Sources/TuistSigning/SecurityController.swift

```diff
@@ -43,21 +46,11 @@ final class SecurityController: SecurityControlling {
 
     // MARK: - Helpers
 
-    private func keyExists(at path: AbsolutePath, keychainPath: AbsolutePath) throws -> Bool {
-        do {
-            try System.shared.run("/usr/bin/security", "find-key", path.pathString, "-P", "", "-k", keychainPath.pathString)
-            logger.debug("Skipping importing private key at \(path.pathString) because it is already present")
-            return true
-        } catch {
-            return false
-        }
-    }
-
-    private func certificateExists(at path: AbsolutePath, keychainPath: AbsolutePath) throws -> Bool {
+    private func certificateExists(_ certificate: Certificate, keychainPath: AbsolutePath) throws -> Bool {
         do {
-            try System.shared.run("/usr/bin/security", "find-certificate", path.pathString, "-P", "", "-k", keychainPath.pathString)
-            logger.debug("Skipping importing certificate at \(path.pathString) because it is already present")
-            return true
+            let existingCertificates = try System.shared.capture("/usr/bin/security", "find-certificate", "-c", certificate.name, "-a", keychainPath.pathString)
+            logger.debug("Skipping importing certificate at \(certificate.publicKey.pathString) because it is already present")
+            return !existingCertificates.isEmpty
```

Comment:
if `existingCertificates.isEmpty == true` then we'd show `Skipping importing certificate` when we should not ... ?